dags/xcom_gcs_ds.py

```diff
@@ -3,25 +3,19 @@
 from airflow.utils.dates import days_ago
 
 from datetime import datetime
-import json
```

Comment:
The `airflow.models import DAG` statement can be removed since the `@dag` decorator is being used.